fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/mv/AbstractSelectMaterializedIndexRule.java

```diff
@@ -98,12 +99,12 @@ protected boolean containAllRequiredColumns(
         OlapTable table = scan.getTable();
 
         Set<String> requiredMvColumnNames = requiredScanOutput.stream()
-                    .map(s -> normalizeName(Column.getNameWithoutMvPrefix(s.getName())))
-                    .collect(Collectors.toSet());
+                .map(s -> normalizeName(Column.getNameWithoutMvPrefix(s.getName())))
+                .collect(Collectors.toCollection(() -> new TreeSet<String>(String.CASE_INSENSITIVE_ORDER)));
 
         Set<String> mvColNames = table.getSchemaByIndexId(index.getId(), true).stream()
                 .map(c -> normalizeName(parseMvColumnToSql(c.getNameWithoutMvPrefix())))
-                .collect(Collectors.toSet());
+                .collect(Collectors.toCollection(() -> new TreeSet<String>(String.CASE_INSENSITIVE_ORDER)));
```

Comment:
i think we should refactor this code later, use nereids struct to present mv's column to avoid string compare